### ISSUE TITLE
A separator to mark the old messages when browsing old messages

### ISSUE THREAD

**Issue by fbm-static** [erming/shout#424]
Hello,

When I click on "Show More" it doesn't have any marker or atleast different color to mark which of the lines were the older ones. It would be nice to make this clear.

Thank you.

**Comment by toXel** [comment]
+1

Something like this red line in Quassel:
![Quassel Screenshot](http://www.linuxlinks.com/portal/content/reviews/Internet/IRC/Screenshot-QuasselIRC.png)

Everytime when you click on another channel the line is added at the bottom of the log. So you can keep track of your last position in a channel very easily.

**Comment by YaManicKill** [comment]
@toXel That's a different feature, more like #382. It shows what messages you had read up to before more came in and you clicked onto a different channel.

I think what @fbm-static wants is to know where the top message was before he loaded more messages from the server from the past.

**Comment by toXel** [comment]
@YaManicKill you're right #382 is more like the thing I want so... just ignore my comment above ;)
